index.js

```diff
@@ -196,12 +199,18 @@ export default class Carousel extends Component {
     for (let i = 0; i < pageLength; i += 1) {
       bullets.push(
         <TouchableWithoutFeedback onPress={() => this._animateToPage(i)} key={`bullet${i}`}>
-          <View style={i === this.state.currentPage ? styles.chosenBullet : styles.bullet} />
+          <View
+            style={i === this.state.currentPage ?
+              [styles.chosenBullet, this.props.chosenBulletStyle] :
+              [styles.bullet, this.props.bulletStyle]}
+          />
         </TouchableWithoutFeedback>);
     }
     return (
       <View style={styles.bullets}>
-        {bullets}
+        <View style={[styles.bulletsContainer, this.props.bulletsContainerStyle]}>
```

Comment:
So we can maintain the absolute position, margins, and centering with the outer `View` and only customize  the "real"  bulletsContainer (inner `View`).

But if you see a better or more elegant aproach, I'll be glad to know about it :)